docs/aws/setup-mainnet-validator-from-scratch.md

```diff
@@ -146,7 +146,7 @@ cd casper-node/
 > installed.
 
 ```
-git checkout release-1.2
+git checkout release-1.3.2
```

Comment:
@mrkara should this not be 1.3.4?

Reply:
It's not a typo. There is no 1.3.4 branch (yet).